lib/Maskings/AttributeMasking.cpp

```diff
@@ -37,22 +37,22 @@ void arangodb::maskings::InstallMaskings() {
 }
 
 std::unordered_map<std::string, ParseResult<AttributeMasking> (*)(
-                                    Path, Maskings*, VPackSlice const&)>
+                                    Path, Maskings*, velocypack::Slice)>
     AttributeMasking::_maskings;
 
 ParseResult<AttributeMasking> AttributeMasking::parse(Maskings* maskings,
-                                                      VPackSlice const& def) {
+                                                      velocypack::Slice def) {
   if (!def.isObject()) {
     return ParseResult<AttributeMasking>(
         ParseResult<AttributeMasking>::PARSE_FAILED,
         "expecting an object for collection definition");
   }
 
-  std::string path = "";
-  std::string type = "";
+  std::string path;
+  std::string type;
```

Comment:
Looks unused
```suggestion
```

Reply:
Both `_type` and `_path` are used later.